tests/interactive_broker_version.py

```diff
@@ -283,6 +302,13 @@ def test_version(version, cmd=None, deploy=True, conf={}, debug=False,
         type=str,
         default=None,
         help='SASL mechanism (PLAIN, SCRAM-SHA-nnn, GSSAPI, OAUTHBEARER)')
+    parser.add_argument(
+        '--oidc-method',
```

Comment:
Should be named oauthbearer-method since oidc is one of the methods.
E.g., "oidc_method == oidc" and "oidc_method == default" are kind of ambiguous.